src/main/scala/ai/privado/utility/ConcurrentProcessor.scala

```diff
@@ -12,7 +12,8 @@ import scala.concurrent.{Await, Future}
 
 object ConcurrentProcessor {
   val zContext = new ZContext()
-  val STOP     = "STOP"
+  zContext.getContext.setMaxSockets(4096)
```

Comment:
@tuxology I guess we should take this value as an input to CLI by setting this value of 4096 as a default value to that input.

Reply:
Updated and tested this with YAML on pay repo. We do need to specific the value in string for now, eg "2048". Default has been set for 4096 and upper limit is 10,000